sonar-project.properties

```diff
@@ -0,0 +1,11 @@
+# The projectKey will need to be updated once we have this on CI
+sonar.projectKey=guolivei-rhub-api
```

Comment:
Not sure that a specific keyname should be included here, probably should be anonymized

Reply:
Fixed on: https://github.com/resource-hub-dev/rhub-api/pull/147/commits/602b1cfaf6649838a04b4d8aadb111a4d6a1c283